modules/account-lib/src/coin/hbar/tokenTransferBuilder.ts

```diff
@@ -0,0 +1,191 @@
+import { BaseCoin as CoinConfig } from '@bitgo/statics';
+import * as Long from 'long';
+import { AccountId, TokenId } from '@hashgraph/sdk';
+import { proto } from '../../../resources/hbar/protobuf/hedera';
+import {
+  BaseKey,
+  BuildTransactionError,
+  InvalidParameterValueError,
+  SigningError,
+  TransactionType,
+} from '@bitgo/sdk-core';
+import { TransactionBuilder, DEFAULT_M } from './transactionBuilder';
+import { Transaction } from './transaction';
+import {
+  isValidAddress,
+  isValidAmount,
+  isValidToken,
+  stringifyAccountId,
+  stringifyTokenId,
+  getHbarTokenFromTokenName,
+} from './utils';
+
+export class TokenTransferBuilder extends TransactionBuilder {
```

Comment:
OK to start, but you may want to subclass `TransferBuilder` if you find a lot of similarities.

CC @noel-bitgo

Reply:
Sorry, wrong Noel.

CC @tunneln

Reply:
I considered using TransferBuilder before, but I need to get `to` and `amount` fields where they are `private` in `TransferBuilder`. Do u think it is fine to change it to `protected` for example?

Reply:
refactored and saved 60 lines, if adding a boolean `isToken` to `initBuilder` and `buildImplementation` is fine